x-pack/plugins/canvas/public/state/reducers/workpad.js

```diff
@@ -24,9 +24,13 @@ import { APP_ROUTE_WORKPAD } from '../../../common/lib/constants';
 export const workpadReducer = handleActions(
   {
     [setWorkpad]: (workpadState, { payload }) => {
-      platformService
-        .getService()
-        .setRecentlyAccessed(`${APP_ROUTE_WORKPAD}/${payload.id}`, payload.name, payload.id);
+      pluginServices
```

Comment:
This should probably be moved into a hook on the route rather than be in the reducer.  Making a note for myself.

Reply:
FYI: `pluginServices.getServices()` has to die... it's still an unsafe access of a module-scope variable, but I'm saving that for a final PR across all of the services, so it's clear that's what's happening.